cypress/integration/embed/share_embed_spec.js

```diff
@@ -0,0 +1,22 @@
+describe('Embedded Queries', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit('/queries/new');
+  });
+
+  it('are shared with text parameters', () => {
```

Comment:
I'm not sure what this is testing?

Reply:
Yeah it's missing an `expect` after `cy.visit(embedUrl);`

Reply:
I guess I was treating it more like a smoke test because it has previously exploded with a 500 when shared. Since there's nothing specific I want to expect here, but merely a "I got here and it shows data", I guess I should snap a Percy?

Reply:
Shouldn't this explode because it uses a text parameter?

Reply:
> I got here and it shows data

So the test is fine as it would fail by default upon status code 500.

**But** if it _should_ fail, you might need to do something like this:
```js
cy.server()
cy.route(embedUrl).as('embedRequest'); // listen to redirect
cy.visit(embedUrl, { failOnStatusCode: false }); // prevent 500 from failing test
cy.wait('@embedRequest').then((xhr) => {
  // expect xhr status code to be 500
});
```

Reply:
Not sure I follow on the xhr stuff here. I'm actually visiting a page here.
Anyway, I changed the existing test to snap a Percy and added another one that doesn't fail on 403 and snaps an additional Percy.